src/_includes/templates/forms/file-a-privacy-complaint-form.html

```diff
@@ -137,38 +141,27 @@
                    required>
         </div>
         <div class="form-l_col form-l_col-1">
-           <label for="request-desc">
-                <span class="form-label-header" for="request-desc">
-                    Information to amend
-                </span>
-                <br>
-                <span class="micro-copy micro-copy__large">
-                    Be as specific as possible and explain why you believe that the information in
-                    question is not accurate, relevant, timely, or complete. Please note that a
-                    requester bears the burden of proving by the preponderance of the
-                    evidence that information is not accurate, relevant, timely, or complete.
-                </span>
+            <label class="form-label-header" for="complaint-desc">
+                Description of complaint
```

Comment:
Sounds good, is there a selector that could add it based on the `required` attribute of the associated form input? Otherwise a class would do. Opened https://github.com/cfpb/cfgov-refresh/issues/927 to continue the discussion.